code/modules/organs/external/_external.dm

```diff
@@ -137,22 +140,25 @@
 	if(module)
 		module.organ_installed(src, owner)
 
-/obj/item/organ/external/removed(mob/living/user, redraw_mob = TRUE)
-	if(!owner)
+/obj/item/organ/external/removed(mob/living/user, redraw_mob = TRUE, mob/living/carbon/human/owner_)
+
+	var/mob/living/carbon/human/victim = owner_ ? owner_ : owner // Used to keep and put into other's removed procs.
```

Comment:
Why we might need to override owner?

Reply:
"Because when head is seperated from the body, it loses it's owner var(as well as the brain). However this entire code is made for the seperated head, which just contradicts itself."